pool.go

```diff
@@ -40,6 +40,25 @@ type ClientConn struct {
 	unhealthy bool
 }
 
+func NewPool(addres string, capacity int) *Pool {
+
+	pool, err := New(func() (*grpc.ClientConn, error) {
+
+		clientConn, err := grpc.Dial(addres, grpc.WithInsecure())
+		if err != nil {
+			// ruokutility.Flog.Error("service err is " + err.Error())
```

Comment:
Dead code should be avoided in the package (especially since `ruokutility` seems to be specific to your application)